### PR TITLE
docs(examples): update basic.html and basic.built.html

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: examples/basic.built.html [hunk 1]
```diff
@@ -1,52 +1,79 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset='utf-8'>
+    <meta charset="utf-8" />
```
[the remark this answers is not in the document]
Comment (marcoscaceres) [reply examples/basic.built.html:4]: Yes, but I'd rather just leave it to prettier, as we can't turn it off.

**Review by sidvishnoi — APPROVED** [review]

**Comment by sidvishnoi** [comment]
Update starter.html while we're here?

**Comment by marcoscaceres** [comment]
Starter spec looks ok, tbh... although it's not particularly useful. I have some ideas of how to make it more useful.  

What we should actually do is move the basic specs into tests, as that's actually what they are used for. 

I'll do that as a followup.

**Comment by ONFU** [comment]
live comende of districk bifg datta of big boX and respown of of my NAuthority am Snake lyv comince of Rampax Orinox and lt's my commendement of the laning lanis patrick maloin of son jule cesard modhosthe am was alway jule cesar at rome and am the spartacus and the ragnard loybroke and am gouverange of siera leone and am paplo rheydheyshen kevin paré

**Pull request merged**